### ISSUE TITLE
[Bug]: Templates are not working

### ISSUE THREAD

**Issue by ziomek64** [MvvmCross/MvvmCross#4867]
### Description

Are the templates on https://github.com/Plac3hold3r/MvxScaffolding official or not ? Because you feature them on the website but they are completely outdated, I can't seem to be able to build it.

### Steps to Reproduce

.

### Version with bug

9.2.0

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Android

**Comment by Cheesebaron** [comment]
Yeah, these are not official templates. Not sure if @Plac3hold3r is planning on updating these or not.

You can use this as a starting point: https://github.com/MvvmCross/MvvmCross-Samples/tree/master/Starter

**Comment by Cheesebaron** [comment]
Also since this is not an issue with the code in this repo, I am converting this to a discussion instead.
